core/com.b2international.snowowl.identity.ldap/src/com/b2international/snowowl/identity/ldap/LdapIdentityPlugin.java

```diff
@@ -29,12 +29,13 @@
 
 	@Override
 	public IdentityProvider create(Environment env, LdapIdentityProviderConfig configuration) throws Exception {
-		return new LdapIdentityProvider(configuration);
+		final LdapIdentityProvider identityProvider = new LdapIdentityProvider(configuration);
+		identityProvider.testLdapSettings();
```

Comment:
It would be great to abstract and pull this up to the `IdentityProvider` with some awesome name later on and use it for all identity provider implementations.

Reply:
I did this; it's not getting much use at the moment, but can be useful for future identity providers that require connection to a remote server.